### PR TITLE
Feature/18 sign up page

### PR DESCRIPTION
**Pull request by SashaBondarchuk**
https://trello.com/c/3U6LEqQS/18-sign-up-page

image
image
image
image
image

created one module for 2 components
image
image

### CODE REVIEW

**Comment by SashaBondarchuk** [comment]
[Sign Up page](https://trello.com/c/3U6LEqQS/18-sign-up-page)

File: frontend/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -11,6 +11,10 @@ const routes: Routes = [
         path: 'main',
         loadChildren: () => import('./modules/main/main.module').then((m) => m.MainModule),
     },
+    {
+        path: 'register',
+        loadChildren: () => import('./modules/sign-up/sign-up.module').then((m) => m.SingUpModule),
+    },
```
Comment (tatianahutii) [inline frontend/src/app/app-routing.module.ts:17]: Please consolidate your work with developer from Log In page and create one module for 2 components

File: frontend/src/app/modules/sign-up/sign-up/sign-up.component.sass [hunk 2]
```diff
@@ -0,0 +1,145 @@
+$mobile: 480px
+$extra-large: 1440px
```
Comment (tatianahutii) [inline frontend/src/app/modules/sign-up/sign-up/sign-up.component.sass:2]: move to a new separate file in frontend/src/styles/.....

File: frontend/src/app/modules/sign-up/sign-up/sign-up.component.sass [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+    font-family: 'Montserrat', sans-serif
+    font-size: 42px
+    line-height: normal
+
+.register-page-container 
+    align-self: start
+    width: 75%
+    max-width: 1078px
+    border-radius: 15px
+    background-color: #222327
```
Comment (tatianahutii) [inline frontend/src/app/modules/sign-up/sign-up/sign-up.component.sass:42]: move to frontend/src/styles/color-variables.sass

File: frontend/src/app/modules/sign-up/sign-up/sign-up.component.sass [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+    border: none
+    outline: none
+    padding: 0 0 0 20px 
+    &:focus
+        outline: none
+
+.hide-password 
+    line-height: normal
+    fa-icon
+        color: #91919380
```
Comment (tatianahutii) [inline frontend/src/app/modules/sign-up/sign-up/sign-up.component.sass:116]: the same

File: frontend/src/styles/color-palette.sass [hunk 5]
```diff
@@ -0,0 +1,7 @@
+:root
+  --color-dark:#1D1E21
+  --color-extra-dark:#121415
+  --color-primary:#705FF5
+  --color-success:#61AD00
+  --color-gray:#777777
+  --color-tangerine:#F5A75F
```
Comment (tatianahutii) [inline frontend/src/styles/color-palette.sass:7]: please move this to frontend/src/styles/color-variables.sass

**Review by tatianahutii — CHANGES_REQUESTED** [review]

File: frontend/src/styles/color-variables.sass [hunk 6]
```diff
@@ -1,13 +1,16 @@
...
-$rich-black:#121415
+$dark-charcoal: #222327
 $cornflower: #705FF5
 $green: #61AD00
+$dark-olive-green: #427302
 $grey: #777777
+$transparent-grey: #91919380
 $light-copper: #F5A75F
 $dark-grey: #121415
+$medium-grey: #222327
```
Comment (tatianahutii) [inline frontend/src/styles/color-variables.sass:12]: color duplication

**Review by tatianahutii — CHANGES_REQUESTED** [review]
But everything else looks really good

**Review by tatianahutii — APPROVED** [review]
LGTM

**Comment by SashaBondarchuk** [comment]
there was merge conflict with some dublication in color-variables.sass and app routing, so I have resolved it

**Review by tatianahutii — APPROVED** [review]

File: frontend/src/app/modules/login/sign-up-page/sign-up.component.ts [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+
+@Component({
+    selector: 'app-sign-up',
+    templateUrl: './sign-up.component.html',
+    styleUrls: ['./sign-up.component.sass'],
+})
+export class SignUpComponent implements OnInit {
+    public showPassword: boolean;
+
+    ngOnInit(): void {
```
Comment (apofis96) [inline frontend/src/app/modules/login/sign-up-page/sign-up.component.ts:11]: Get rid of onInit here

**Review by apofis96 — CHANGES_REQUESTED** [review]

**Review by tatianahutii — APPROVED** [review]

**Pull request merged**